.cicd/eosio-contracts-unittest.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+
+[[ ! -z "$CONTRACTS_VERSION" ]] || export CONTRACTS_VERSION="$(cat "$PIPELINE_CONFIG" | jq -r '.dependencies["eosio.contracts"]')"
+git clone -b "$CONTRACTS_VERSION" https://github.com/EOSIO/eosio.contracts.git 
+
+# Docker Commands
+ARGS=${ARGS:-"--rm --init -v $(pwd):$MOUNTED_DIR"}
+. $HELPERS_DIR/docker-hash.sh
+
+if [[ $BUILDKITE == true ]]; then
```

Comment:
I think this `if` statement should be removed. We only use this `[[ $BUILDKITE == true ]]` guard to prevent `buildkite-agent` from being invoked when running the CI code on our local machines, but `buildkite-agent` is not being used in this script.

Reply:
Done.